rclcpp_components/package.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>rclcpp_components</name>
+  <version>0.6.2</version>
+  <description>Package containing container tools for dynamically loadable components</description>
```

Comment:
Drop `container`? or reword to be something like `containing tools and containers for ...`?

Reply:
Also punctuation? I guess it's a sentence fragment.